Makefile

```diff
@@ -213,6 +218,11 @@ else
             IP_DIR=$(IP_DIR)
 endif
 
+$(FPGA_BIN_FILE): $(FPGA_FILE)
+	cd $(FPGA_BUILD_DIR) && \
+        echo -e "all:\n{\n    $(FPGA_FILE)\n}\n" > bs.bif && \
+        $(BOOTGEN) -image bs.bif -arch zynq -process_bitstream bin
+
```

Comment:
Is there anything special about the format of the bin file? Presumably it is just the bitstream with the header removed. We can tell bitgen to write out the bin file instead of/as well as the bit file.

Reply:
The only thing special about the bin file is that it is required by the FPGA manager